traffic_ops/traffic_ops_golang/deliveryservice/servers/servers.go

```diff
@@ -1023,6 +1027,7 @@ func scanDSInfoRow(row *sql.Row) (DSInfo, bool, error) {
 		}
 		return DSInfo{}, false, fmt.Errorf("querying delivery service server ds info: %v", err)
 	}
+	di.Active = active == tc.DSActiveStateActive
```

Comment:
I am so lost here, why is a boolean being assigned a string and then use a comparison operator?